packages/adapter-nuxt2/src/hooks/project-init.ts

```diff
@@ -38,6 +38,10 @@ const configurePrismicModule = async ({
 }: ConfigurePrismicModuleArgs) => {
 	let nuxtConfigPath = helpers.joinPathFromRoot("nuxt.config.js");
 
+	const endpoint =
```

Comment:
you can solely rely on `repositoryName` now, it ensures that we add `.cdn` to the endpoint

Reply:
Not with Nuxt 2, it's still running `@prismicio/client@5`